frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -168,42 +168,67 @@ export const PrometheusUtilizationItem = withDashboardResources<PrometheusUtiliz
     utilizationQuery,
     totalQuery,
     duration,
+    adjustDuration = (start: number) => start,
     title,
     TopConsumerPopover,
     humanizeValue,
     byteDataType,
     setTimestamps,
+    isDisabled = false,
   }) => {
+    let stats = [];
+    let utilization: PrometheusResponse, utilizationError: any;
+    let total: PrometheusResponse, totalError: any;
+    let max: DataPoint<number>[];
+    let isLoading = false;
+
+    const effectiveDuration = React.useMemo(
```

Comment:
with respect to the comment above, we will need to change this to



```
const effectiveDuration = React.useMemo(
  () => adjustDuration ? adjustDuration(UTILIZATION_QUERY_HOUR_MAP[duration]) :UTILIZATION_QUERY_HOUR_MAP[duration] ,
  [adjustDuration, duration],
);
```

Reply:
done